### ISSUE TITLE
"Finish early" does not appear till I have one correct answer

### ISSUE THREAD

**Issue by szabgab** [kantord/LibreLingo#1804]
When I start a new session by clicking "Practice Sentences" I only have a "Skip" button at the bottom.
The "Finish early" button only appears once I had at least one correct answer.

I would like to be able to "Finish Early" any time in the course.

Basically I'd like to be able to cancel the current session immediately when I started it.

**Comment by kantord** [comment]
This is basically intentional. Finish early doesn't cancel the session but saves a partially finished one as it is. Without having at least one correct answer there's no progress to save according the current logic.

We could make this button appear also when you don't have any correct results, but then it would probably have to lead back to the course screen without saving anything (as there's no progress to save)

Or perhaps it's should be a different button? 🤔

**Comment by szabgab** [comment]
Yeah, maybe an X at the top-left corner of the screen should allow the cancellation of the current practice session. If the user already made progress it should ask for confirmation.
